### ISSUE TITLE
not all clean-css options do not seem to work

### ISSUE THREAD

**Issue by ljosberinn** [olback/es6-css-minify#26]
Hi,

I have a CSS file importing 3 others, in the same folder. Minifying them automatically removes all @import statements.

According to Marketplace description, you should be able to assign custom clean-css rules via `.cleancssrc` or settings:

```sh
"es6-css-minify.cleancssConfigFile": ".cleancssrc",
  "es6-css-minify.css": {
    "inline": ["all"]
  }
```

A seperate `.cleancssrc` file containing solely

```sh
{
  "inline": ["all"]
}
```
does not help either.

After each change, Minify config was reloaded.

None of these changes lead to local files being imported. At best, a vague message "Minified. x.y% of original but with errors" appears, errors cannot be found anywhere apparently though (no console.log entry to be found).

Am I simply doing it wrong or is this currently broken?

**Comment by olback** [comment]
Working on it.

**Comment by olback** [comment]
Should be fixed in 2.0.0.

**Comment by olback** [comment]
Update published.
